spec/support/capybara.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.configure do |config|
+  config.before :each, type: :system do
+    driven_by :rack_test
```

Comment:
Whilst you're at it can you prune this line from any existing system specs as it's no longer needed.